layouts/Error.tsx

```diff
@@ -0,0 +1,66 @@
+// CSS
+import styles from '../styles/layouts/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Footer from '@/components/Footer';
+
+
+export default function ErrorLayout({
+  children,
+}:{
+  children: React.ReactNode
+}) {
+  const [allowReport, setAllowReport] = useState(false);
+  const departureUrl = 'https://tcsca';
+  const returnHref = '/'; // Allow user to return to home
+  const reportHref = 'https://github.com/Trent-Computer-Science-Club-Association/ClubWebsite/issues/new?labels=page-error&template=page-error.md'; // Allow users to report on GitHub
+  
+  useEffect(() => {
+    // If the user is coming from a broken link on our site, they can report an issue
+    // Note: document.referrer is not always reliable as there are ways it can be blocked
+    if (document.referrer.includes('localhost:3000' || document.referrer.startsWith(departureUrl))) {
+     setAllowReport(true);
+    }
```

Comment:
Can you run `npm run format` and `npm run lint` I don't think your formatter is working you can find the extension debug log in vscode under the output window (where the terminal is), and then prettier-eslint. (We can discuss in discord if its still giving problems).